orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandlerTest.kt

```diff
@@ -114,7 +109,15 @@ object RunTaskHandlerTest : SubjectSpek<RunTaskHandler>({
   val contextParameterProcessor = ContextParameterProcessor()
   val dynamicConfigService: DynamicConfigService = mock()
   val taskExecutionInterceptors: List<TaskExecutionInterceptor> = listOf(mock())
-  val stageResolver = DefaultStageResolver(StageDefinitionBuildersProvider(emptyList()))
+  val stageResolver : DefaultStageResolver = mock()
+  val manualJudgmentStage : ManualJudgmentStage = ManualJudgmentStage()
+
+  whenever(stageResolver.getStageDefinitionBuilder(manualJudgmentStage.type, null)) doReturn manualJudgmentStage
+  whenever(stageResolver.getStageDefinitionBuilder(zeroTaskStage.type, null)) doReturn zeroTaskStage
```

Comment:
Is this required?  My credibility is potentially shot, but I took out this line and the tests still work.

Reply:
Not required, was part for the previous test implementation, I removed it now